Makefile

```diff
@@ -20,6 +20,7 @@ lint:
 
 format:
 	yarn format
+	pipenv run isort **/*.py application.py
```

Comment:
The lint command should also be updated so that the sort order can be enforced. The isort docs mention the `--diff` flag which could be used for this.

Reply:
The `run_lint_python.sh` file contains the `isort --check` command and uses the exit code to determine success or failure. This is the same pattern we follow for the other lint commands for our python code.